src/Exercise9/Exercise9PaintCalculator.java

```diff
@@ -1,17 +1,13 @@
 package Exercise9;
 
 public class Exercise9PaintCalculator {
-    private final static double CONVERSION_FACTOR = 0.002857142857143;
+    private final static double SQUARE_FEET_PER_GALLON = 350;
 
     public String paintNeededForRectangularRoom(double width, double length) {
-        if (length * width % 350 == 0) {
-            return "" + (int) (length * width / 350) + " gallons of paint needed";
-        } else return (int) Math.ceil(length * width * CONVERSION_FACTOR) + " gallons of paint needed";
+        return (int) Math.ceil(length * width / SQUARE_FEET_PER_GALLON) + " gallons of paint needed";
```

Comment:
Think we already spoke about this, but looks like you can simplify the logic somewhat if you change the way you store the conversion factor.